examples/daal/cpp_sycl/source/k_nearest_neighbors/bf_knn_dense_search_batch.cpp

```diff
@@ -0,0 +1,170 @@
+/* file: bf_knn_search_dense_batch.cpp */
```

Comment:
We need not just indices search, but regression example! It should demonstrate how to solve regression problem: compute response from the closest observations.

I suggest to add separate example

Reply:
Sure. Thanx for relevant comment

Reply:
Such example is added. Please check `bf_knn_dense_regression_batch.cpp`